pkgs/development/python-modules/pyosmium/default.nix

```
@@ -64,6 +57,8 @@ buildPythonPackage rec {
   nativeCheckInputs = [
     pytestCheckHook
     shapely
+    werkzeug
+    pytest-httpserver
```

Alphabetical order is usually preferred.
```suggestion
    pytest-httpserver
    pytestCheckHook
    shapely
    werkzeug
```